templates/server/post-receive.erb

```diff
@@ -111,12 +111,33 @@ $stdin.each_line do |line|
         end
         do_cmd("git submodule sync")
         do_cmd("git submodule update --init --recursive")
+        <% if @git_repo_r10k -%>
+        if File.exists? 'Puppetfile'
```

Comment:
`File.exists?` is deprecated in favor of `File.exist?`. Ruby 3.2 removed the deprecated variant.
```suggestion
        if File.exist? 'Puppetfile'
```

And yes, I now see we do the same thing earlier in the file.

Reply:
Fixed for both.